devilizer/visualizer/display_items.py

```diff
@@ -0,0 +1,23 @@
+def display_items(in_lst: list, current: int) -> str:
+    """Display a list of values.
+
+    Args:
+        in_lst (list): The list of integers to display.
+        current (int): The current column being iterated over.
+
+    Returns:
+        str: Display of values.
+    """
+    result = ""
+    max_height = max(in_lst)
+    for i in reversed(range(0, max_height+1)):
```

Comment:
This range can be reversed directly
```suggestion
    for i in range(max_height, -1, -1):
```